plugins/modules/mas.py

```diff
@@ -299,3 +307,12 @@ def main():
 
 if __name__ == '__main__':
     main()
+
+
+def is_version_greater(version, reference_version):
+    version_parts = [int(part) for part in version.split('.')]
+    reference_parts = [int(part) for part in reference_version.split('.')]
+    if version_parts > reference_parts:
+        return True
+    else:
+        return False
```

Comment:
Instead of coding a function for that here, it would arguably make more sense to reuse existing code for that. Take a look at how
https://github.com/ansible-collections/community.general/blob/main/plugins/filter/version_sort.py
works, that could certainly help.

Reply:
Good spot. I also realised that `LooseVersion` is already getting imported hence this can be used to compare the versions.